unitq/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "7781542f-b4a2-40e2-86cd-9987980a0ead",
+  "app_id": "unitq",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Harness the power of user feedback to improve product quality.",
+    "title": "unitQ",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::macOS",
+      "Supported OS::Windows",
+      "Category::Monitoring",
+      "Category::Metrics"
+    ]
+  },
+  "author": {
+    "name": "unitQ",
+    "homepage": "https://github.com/DataDog/integrations-extras",
```

Comment:
Do you have another website for unitq?

Reply:
Yes updated it. All the other plugins have this URL btw.